ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```
@@ -463,6 +463,71 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-single-node,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build02
+    context: ci/prow/e2e-aws-single-node-graceful-shutdown
+    decorate: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-machine-config-operator-master-e2e-aws-single-node-graceful-shutdown
+    rerun_command: /test e2e-aws-single-node-graceful-shutdown
```

```suggestion
    optional: true
    rerun_command: /test e2e-aws-single-node-graceful-shutdown
```
This should not be blocking

---

optional set to true is not blocking, right? [doc](https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/)

---

yep that's why i added it :)